ui/pages/settings/settings-search/settings-search.js

```diff
@@ -34,7 +34,7 @@ export default function SettingsSearch({
   ///: END:ONLY_INCLUDE_IN
   const settingsSearchFuse = new Fuse(settingsRoutesListArray, {
     shouldSort: true,
-    threshold: 0.2,
+    threshold: 0.3,
```

Comment:
Q: where did we get 0.3 from? I wonder why we've lowered the threshold from the default value of 0.6

Reply:
Thanks for reviewing ! As 0 is the perfect match and we went for 0.2, so 0.3 is the next best matching algorithm .
For the default value 0.6, this is the result and it is too vague for us :( 

<img width="536" alt="Screenshot 2023-03-06 at 15 15 13" src="https://user-images.githubusercontent.com/12678455/223151567-7b6abb0c-161b-485a-a19b-a947641a364f.png">

<img width="528" alt="Screenshot 2023-03-06 at 15 16 39" src="https://user-images.githubusercontent.com/12678455/223151733-2c0cf95d-07a4-4cfc-8a46-1477fa7e0f4f.png">


I tested also for single match and multiple results (network for example),0.3 seems to be a good value to give.